system_strings/sysmes_text.en

```diff
@@ -5861,8 +5863,8 @@ I feel bad about deceiving Senpai, but I'd made a promise with Arcueid first.
 // あの人 both Ciel here, not arc.
 // 別の話 presumably referring to 尊敬と、好き嫌いは別の話だ。
 // @flowchart_descr
-Ciel was the one who saved my life.
- Ciel was the one who protected the city.
+Ciel-senpai was the one who saved my life.
+ She was the one who protected the city.
```

Comment:
Maybe `She was also the one` just to make it clear they're both referring to Ciel.